axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/app/AppBusinessProjectServiceImpl.java

```diff
@@ -51,4 +57,28 @@ public AppBusinessProjectServiceImpl(
   public AppBusinessProject getAppBusinessProject() {
     return appBusinessProjectRepo.all().fetchOne();
   }
+
+  @Override
+  public Unit getDaysUnit() throws AxelorException {
+    if (Objects.isNull(getAppBusinessProject().getDaysUnit())) {
+      throw new AxelorException(
+          getAppBusinessProject(),
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          I18n.get(BusinessProjectExceptionMessage.PROJECT_CONFIG_DEFAULT_HOURS_PER_DAY_MISSING));
+    }
+
+    return getAppBusinessProject().getDaysUnit();
+  }
+
+  @Override
+  public Unit getHoursUnit() throws AxelorException {
+    if (Objects.isNull(getAppBusinessProject().getHoursUnit())) {
+      throw new AxelorException(
+          getAppBusinessProject(),
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          I18n.get(BusinessProjectExceptionMessage.PROJECT_CONFIG_DEFAULT_HOURS_PER_DAY_MISSING));
+    }
+
+    return getAppBusinessProject().getHoursUnit();
```

Comment:
Store getAppBusinessProject().getDaysUnit() and getAppBusinessProject().getHoursUnit() in a variable